src/styles/blocks/card.scss

```
@@ -0,0 +1,95 @@
+
+.main__container {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 50px 0;
+  max-width: 1100px;
+  justify-content: center;
+  margin: 0 auto;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  margin: 0 48px 46px 0;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: flex;
+    align-self: center;
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+  
+  &__name {
+    @include font-parameters ($fontFamily, 500, normal);
+
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+    margin: 0 0 4px;
+  }
+  
+  &__code {
+    @include font-parameters ($fontFamily, 400, normal);
+
+    font-size: 10px;
+    line-height: 14px;
+    color: $colorGray;
+    margin: 0 0 16px;
+  }
+  
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin: 0 0 16px;
+  }
+  
+  &__price-word {
+    @include font-parameters ($fontFamily, 400, normal);
+
+    font-size: 12px;
+    margin: 0;
+    line-height: 18px;
+    color: $colorGray;
+  }
+  
+  &__price-amount {
+    @include font-parameters ($fontFamily, 700, normal);
+
+    font-size: 16px;
+    line-height: 18px;
+    color: #060b35;
+    margin: 0;
+  }
+  
+  &__button {
+    @extend %flexAlignCenter;
+    justify-content: center;
+    @include font-parameters ($fontFamily, 700, normal);
+    background-color: $colorBlue;
+    height: 40px;
+    border-radius: 5px;
+    border: none;
+    color: #fff;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+    cursor: pointer;
+  }
+  
+  &__button:hover {
+    color: $colorBlue;
+    background-color: #fff;
+    border: 1px solid $colorBlue;
```

Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover
```suggestion
    border-color: $colorBlue;
```